Statistics.py

```diff
@@ -0,0 +1,102 @@
+# Statistics.py
+# Computes statistics and creates cost histogram
+# Ran Libeskind-Hadas, June 2020
+
+# Performs Monte Carlo simulation to compute empirical p-value and histogram
+# of solution costs.
+# https://www.ncbi.nlm.nih.gov/pmc/articles/PMC379178/
+
+from clumpr import DTLReconGraph
+import random
+import matplotlib
+matplotlib.use('tkagg') # need this so plt.show() works
+import matplotlib.pyplot as plt
+
+from typing import List, Dict, Tuple
```

Comment:
The `List`, `Dict`, and `Tuple` type from typing is used for defining nested types (say, `Dict[str, int]`, `Tuple[str, int]`). It seems that you are not using these in your code, so feel free to remove this line. Note that if you are only using `dict`, `float` and other basic python types for type checking, you don't need to import `typing`.